ramls/marccat.raml

```diff
@@ -150,3 +151,28 @@ resourceTypes:
           body:
             text/plain:
               example: "Error in validation"  
+
+  /marccat/auth-header-types:
```

Comment:
Sorry, it is true

Reply:
Again, can you elaborate a bit more here? Are you saying the API should really be `http://okapi/marccat/marccat/auth-header-types` instead of `http://okapi/marccat/auth-header-types`? The code appears to set up the route with one "marccat" in the path. This file is used to generate documentation here: https://s3.amazonaws.com/foliodocs/api/mod-marccat/marccat.html. It is important to get it right. I believe you need to drop the "/marccat" here because you have it nested under a "/marccat" already on line 32.

Reply:
Sorry, I have confused. It is a Get API and the structure is `http://okapi/marccat/auth-header-types`